packages/website/pages/docs/guides/prover-claim-ttko.mdx

```diff
@@ -0,0 +1,39 @@
+import { Callout, Steps } from "nextra-theme-docs";
+
+# Prover Claim TTKO
+
+## Overview
+
+This guide will help you claim TTKO earned as a prover.
+
+## Steps
+
+<Steps>
+
+### View TTKO prover address balance
+
+Go to the Taiko L1 contract read page and select [getTokenTokenBalance(addr address)](https://sepolia.etherscan.io/address/0x6375394335f34848b850114b66A49D6F47f2cdA8#readProxyContract#F10)
+to lookup your TTKO claim balance for your prover's address. 
+
+### Withdraw your TTKO
```

Comment:
maybe can expand this with two steps instead:
1. "Select withdrawTaikoToken and connect wallet
2. "Withdraw TTKO" -> this one should ultimately say "click the "Write"" button at the end

but basically just need that final click button step.

Reply:
Expanded into 2 steps now.